package.json

```diff
@@ -80,6 +80,7 @@
     "@lezer/common": "~1.2.1",
     "@lezer/generator": "~1.7.1",
     "@lezer/lr": "~1.4.2",
+    "@marijn/find-cluster-break": "~1.0.2",
```

Comment:
I don't know why this package is needed. It's not used anywhere and it works for me without it. Sounds like an issue on your end.